apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationCommentList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import django from 'django'
+
+import ModerationComment from './ModerationComment'
+
+export default function (props) {
+  const byText = django.gettext('By ')
+
+  return (
+    <div className="row mb-2">
```

Comment:
We could handle the list here (loop/map).

Reply:
In case we keep the functional component here, we could give it a name and export it afterwards. Something like:

`function moderationCommentList (props) { ... }` or `const moderationCommentList = (props) => { ... }`

then

`export default moderationCommentList`

PS: i know it looks strange with the small caps name, but when importing we can do `ModerationCommentList`. At least we have a name for the function, no?

Reply:
@khamui so we use camel case for functional components and pascal for class components?

Reply:
ok after swapping this I found our linting will not allow unused variables and when we don't need export it then adding a name goes against linting so what i'll do is add this to our design learnings and maybe we could consider a linting update if we feel it's too unreadable.